content/en/continuous_integration/dora_metrics/_index.md

```diff
@@ -0,0 +1,292 @@
+---
+title: DORA Metrics
+kind: documentation
+description: Learn how to use DORA metrics to improve and measure software development.
+is_beta: true
+further_reading:
+- link: "/continuous_integration/pipelines"
+  tag: "Documentation"
+  text: "Learn about Pipeline Visibility"
+- link: "/tracing/service_catalog"
+  tag: "Documentation"
+  text: "Learn about the Service Catalog"
+- link: "https://github.com/DataDog/datadog-ci"
+  tag: "Source Code"
+  text: "Learn about the `datadog-ci` CLI tool"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="https://forms.gle/Eqq6uXfGjYxmqpjDA" header="false" >}}
+DORA Metrics are in private beta for <a href="https://docs.datadoghq.com/continuous_integration/pipelines/">CI Pipeline Visibility</a> customers. To request access, complete the form.
```

Comment:
"DORA Metrics are in private beta for CI Pipeline Visibility, [Test Visibility](https://docs.datadoghq.com/continuous_integration/tests/), and/or [Static Analysis](https://docs.datadoghq.com/continuous_integration/static_analysis/?tab=circleciorbs) customers."